### ISSUE TITLE
404 error page - headings and search input

### ISSUE THREAD

**Issue by yesenia-gtz** [Office-of-Digital-Services/California-State-Web-Template-Website#969]
[404 error page](https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/components/accordions.html)

- [ ] The search bar that links from the 404 page is prefilled with “undefined”, why not just leave the bar blank or have “enter search” 
- [ ] A11y: Current heading structures are H1 and H3. Need to add an H2
- [ ] (per @hamiltonryan )“To find what you need, you can:” should be in larger, lead text font.

**Comment by sharon-rabang** [comment]
@kkoryaka Design reviewed and good to go, thanks!
